app.php

```diff
@@ -1,69 +1,13 @@
 #!/usr/bin/env php
```

Comment:
Could you move this file into a `bin` directory and rename it to `docker-hostmanager`?
No need to add a namespace.

Like https://github.com/iamluc/buse/blob/master/bin/buse